### DISCUSSION TITLE
Interest in Eye Tracking Data Visualization Tool for GSOC'24

### DISCUSSION THREAD

**Discussion by siddanth-6365** [ruxailab/RUXAILAB#364] [General]
Hey @marcgc21 @KarinePistili (sorry for tagging)

I wanted to express my interest in the Eye Tracking Data Visualization Tool project for GSOC'24. I'm Siddanth, currently pursuing my undergraduate degree in Computer Science in India. I've recently completed an internship and served as a teaching assistant in the web development field, where I've gained valuable experience working on various projects. I believe this project aligns perfectly with my skills and interests.

Before diving into the proposal, I took the initiative to explore the GitHub repository (https://github.com/ruxailab/web-eye-tracker-front & https://github.com/ruxailab/web-eye-tracker) to get a better understanding of the project scope. If this is indeed the correct repository, I'm eager to start familiarizing myself with the codebase and understanding how I can contribute effectively. (i guess i am exploring the right repo please confirm)

Furthermore, I'm fully prepared to learn any additional technologies required for the project and to go the extra mile to ensure its success. If there are specific technologies or tools I should focus on, please let me know. I'm already comfortable with various JavaScript frameworks and libraries and enthusiastic about expanding my skills further.

Thank you

**Comment by marcgc21** [comment]
Hi @siddanth-6365 sorry for the late reply but as we indicated before we are handling the conversation with the development community on the Discord channel. 
Feel free to reach me there and see if you are still interested in. We still have plenty of work to do with the eye-tracking system and I believe that you might be possible to work on that.

**Reply by siddanth-6365** [reply]
Hey @marcgc21,

I’m definitely interested in working on this! I’ll reach out to you on Discord and start looking into the new issues. There seems to be plenty to contribute to, so I’ll get started.

Thanks for getting back to me!
